BuildResidentialHPXML/measure.rb

```diff
@@ -3240,10 +3240,10 @@ def self.create_geometry_envelope(runner, model, args)
     end
     return false if not success
 
-    success = Geometry.create_windows_and_skylights(runner: runner, model: model, **args)
+    success = Geometry.create_doors(runner: runner, model: model, **args)
     return false if not success
 
-    success = Geometry.create_doors(runner: runner, model: model, **args)
+    success = Geometry.create_windows_and_skylights(runner: runner, model: model, **args)
```

Comment:
Changes the order that doors/windows are added to the model; the window code is "aware" of other subsurfaces but the door code is not, so the door code needs to come first.